docker-compose.yml

```diff
@@ -27,7 +27,7 @@ services:
                         - /var/run:/var/run
                 environment:
                         - HOME=/cert-home
-                        - NDN_LOG=ndn.Face=ALL
+                        - NDN_LOG=ndncert.*=INFO
```

Comment:
don't change this one. You haven't touched the discovery daemon, which doesn't have any other logging (and definitely not anything from ndncert)